pkg/operator/api/userconfig/apis.go

```diff
@@ -119,17 +119,23 @@ func (api *API) Validate() error {
 	if yaml.StartsWithEscapedAtSymbol(api.Model) {
 		api.ModelFormat = TensorFlowModelFormat
 	} else {
-		if api.ModelFormat == UnknownModelFormat {
-			return errors.Wrap(cr.ErrorMustBeDefined(), Identify(api), ModelFormatKey)
-		}
-
 		if !aws.IsValidS3Path(api.Model) {
 			return errors.Wrap(ErrorInvalidS3PathOrResourceReference(api.Model), Identify(api), ModelKey)
 		}
 
 		if ok, err := aws.IsS3PathFileExternal(api.Model); err != nil || !ok {
 			return errors.Wrap(ErrorExternalNotFound(api.Model), Identify(api), ModelKey)
 		}
+
+		if api.ModelFormat == UnknownModelFormat {
```

Comment:
nit: is there a reason to move this below the two checks above? Generally I like to do the "simpler" / "less expensive" checks first, and checks that modify state first (in case that state is used downstream). Not a big deal though

Reply:
yeah good idea